pkg/service/storage/azure.go

```diff
@@ -20,7 +20,7 @@ func (a *AzureStorageAccessor) supports(storage model.Storage) bool {
 }
 
 func (a *AzureStorageAccessor) createReader(
-	ctx context.Context, storage model.Storage, path string, isFile bool, filter Validator,
+	ctx context.Context, storage model.Storage, path string, isFile bool, filter Validator, from string,
```

Comment:
may be name it `startFrom` ?  (and in all other places)

Reply:
maybeEven startScanFrom ?

Reply:
yep it would be great, because `from` is to abstract